pkg/util/iptables/iptables.go

```diff
@@ -503,6 +518,18 @@ func getIPTablesWaitFlag(vstring string) []string {
 	}
 }
 
+// Checks if iptables-restore has a "wait" flag
+func getIPTablesRestoreWaitFlag(exec utilexec.Interface) restoreWaitFlag {
+	glog.V(4).Infof("checking if iptables-restore can wait")
+	bytes, err := exec.Command(cmdIPTablesRestore, "--help").CombinedOutput()
```

Comment:
@cmluciano no, 1.6.1 was in January 2017, while the wait patches landed in March.  So I assume 1.6.2 and later.